ui/components/component-library/icon/icon.types.ts

```diff
@@ -171,7 +171,7 @@ export enum IconName {
   Bold = 'bold',
   CircleX = 'circle-x',
   Download = 'download',
-  File = 'file',
+  FileIcon = 'file',
   Flask = 'flask',
```

Comment:
![Screenshot 2024-01-25 at 10 49 03 AM](https://github.com/MetaMask/metamask-extension/assets/2182307/6ac8074e-af9a-4dba-9163-58db9e906945)

I renamed the variable to fix lint issue cc @georgewrmarshall